sdk/core/core-amqp/package.json

```diff
@@ -83,7 +83,8 @@
     "rhea-promise": "^1.1.0",
     "tslib": "^2.0.0",
     "url": "^0.11.0",
-    "util": "^0.12.1"
+    "util": "^0.12.1",
+    "@azure/core-asynciterator-polyfill": "^1.0.0"
```

Comment:
fyi @richardpark-msft, @chradek, @HarshaNalluru

Reply:
I wonder if we can skip this as Event Hubs does not have any iterators...
Service Bus has a dependency on core-http, so it will get this for free

@chradek, Can you confirm?

Reply:
Service Bus already imports this though.

Reply:
@ramya-rao-a  Eventhubs already have this dependency. I do not see a reason now to keep it. So, removed it in the latest commit